ntoskrnl/ke/i386/v86vdm.c

```diff
@@ -303,7 +311,15 @@ KiVdmOpcodeIRET(IN PKTRAP_FRAME TrapFrame,
     V86EFlags = EFlags;
 
     /* Check for VME support */
-    ASSERT(KeI386VirtualIntExtensions == FALSE);
+    if (KeI386VirtualIntExtensions)
+    {
+        if (EFlags & EFLAGS_INTERRUPT_MASK)
+        {
+            EFlags |= EFLAGS_VIF;
+            /* Don't set the interrupt flag */
+            V86EFlags &= ~EFLAGS_INTERRUPT_MASK;
```

Comment:
Reviewing again the code, I still have my doubts about this one (which was my suggestion in the previous PR)...

Reply:
just looking at it i think we'd need to handle it here. I.e i think removing the interrupt mask from the flag should be removed.